aws-refresh-login.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# Before running this script you must authenticate through Väylä SAML:
+#
+# python3 aws/login/vaylaAssumeRoleAWSCLI.py --username <Your Väylä username> --account 783354560127 --role ViiteAdmin --region eu-west-1
+#
+python3 aws/login/vaylaAssumeRoleAWSCLI.py --refresh true
```

Comment:
python3 sijasta windowsissa käytetään ilmeisesti komentoa py

Reply:
Yes, this python executable name might differ a bit.